nuxeo-common/src/main/java/org/nuxeo/common/utils/FileUtils.java

```diff
@@ -650,4 +648,15 @@ public static boolean areFilesContentEquals(String expected, String source) {
         }
     }
 
+    /**
+     * Replace illegal characters (: \ / * ? | < >) in a filename with "_"
+     *
+     * @param filename the filename to sanitize
+     * @return the filename with underscores instead of illegal characters
+     * @since 9.1
+     */
+    public static String sanitizeFilename(String filename) {
+        return filename.replaceAll("[:\\\\/*?|<>]", "_");
```

Comment:
Please make this consistent with FileUtils#getCleanFilename from ui-web module (so that ui-web relies on this new method maybe)